selfdrive/car/toyota/values.py

```diff
@@ -1684,6 +1685,8 @@ class CAR:
   CAR.ALPHARD_TSS2: dbc_dict('toyota_nodsu_pt_generated', 'toyota_tss2_adas'),
 }
 
+# These cars have non-standard EPS torque scale factors. All others are 73
+EPS_SCALE = defaultdict(lambda: 73, {CAR.PRIUS: 66, CAR.COROLLA: 88, CAR.LEXUS_IS: 77, CAR.LEXUS_RC: 77, CAR.LEXUS_CTH: 100})
```

Comment:
just use a normal dict, then do `EPS_SCALE.get(car, 100)`

Reply:
Sure, just means we define it in two places if that's okay.

Reply:
actually, think you're right. the defaultdict is cleaner